x-pack/test_serverless/functional/test_suites/observability/screenshot_creation/response_ops_docs/cases/list_view.ts

```diff
@@ -55,15 +55,11 @@ export default function ({ getPageObject, getPageObjects, getService }: FtrProvi
         roleAuthc
       );
       caseIdMonitoring = caseMonitoring.id;
+      await pageObjects.svlCommonPage.loginWithRole('admin');
```

Comment:
I moved it to `before` hook, since the role is not changing and it shouldn't change the logic doing it before each `it` function